app/styles/nypr-ui/_bricks.scss

```diff
@@ -316,15 +316,34 @@
     }
   }
 }
-
+.brick-row--blowout .brick__item .item {
+  &__icons {
+    pointer-events: none;
+    &.play-button{
```

Comment:
Looks like we can remove this since we're not doing the play button right now

Reply:
deleted